dmutils/__init__.py

```diff
@@ -3,4 +3,4 @@
 
 import flask_featureflags
 
-__version__ = '21.0.0'
+__version__ = '22.0.0'
```

Comment:
If this is a major version (breaking) change it would be good to have a CHANGELOG entry to explain what has broken and how to fix apps that use it.

I'm not sure this is a breaking change though? In which case go to 21.1.0 for a new non-breaking feature.

(See the summary of version numbering at http://semver.org/)